chopper/lib/src/response.dart

```diff
@@ -71,10 +71,10 @@ class Response<BodyType> {
   /// Returns the response body as bytes ([Uint8List]) provided the network
   /// call was successful, else this will be `null`.
   Uint8List get bodyBytes =>
```

Comment:
Can you update documentation here ?

Reply:
Actually I don't know about this one

if the original `final BodyType? body;` can be nullable
`Uint8List get bodyBytes` and `String get bodyString` should also be nullable

@JEuler @stewemetal any thoughts about this ?

Reply:
Response class from package:http/src/response.dart has next implementation:

`final Uint8List bodyBytes;`

Reply:
Indeed so maybe just update the document saying we can return null value :)